ui/Screen/TransactionDetails.svelte

```diff
@@ -22,7 +22,9 @@
   if (session.identity !== null) {
     identity = session.identity;
   }
-  const store = fetch(params.id);
+
+  $: payer = transaction.formatPayer(identity);
```

Comment:
Can one's `identity` pay for transactions? Does this need to be a `user`?

Reply:
The formatting needs avatar information and therefore relies on identity. This makes sense I think.

Reply:
Ok, since we haven't implemented wallets yet I guess that makes sense. I've been operating under the assumption that anything registry-related depends on a `user`, and the avatar might come from an `identity` attested to by that `user`.